src/extension.ts

```diff
@@ -181,7 +185,7 @@ export function activate(context: vscode.ExtensionContext) {
 
 					data['completions'].forEach(this_item => {
 						words.push(this_item['trigger']);
-						let item = new vscode.CompletionItem(this_item['trigger'], vscode.CompletionItemKind.Function);
+						let item = new vscode.CompletionItem(this_item['trigger']);
```

Comment:
why dont you want to use the Function/Text completion model?

Reply:
It's been a long time, so I don't remember exactly, but I think I wasn't able to compile the file without removing it. I would have to check

Reply:
ah it could be the api version, what vscode version are you on?